code-challenges.js

```diff
@@ -57,12 +102,35 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const q4 = (array1, var1) => {
+    for (let i = 0; i < array1.length; i++) {
+        let var2 = array1.lastIndexOf(var1)
+        return var2
+    }
+    
+}
+console.log(q4(myNumbers, givenValue1)) // output: 7
+console.log(q4(myNumbers, givenValue2)) // output: 8
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Describe your process:
+    // create function q5
+    // add parameter var1
+    // within function, create variable var2 = var1 sorted then reversed with built-in methods. 
+    // return var2
+    // when invoked, function should display given variables, or any numbers array, in descending order of values.
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+const q5 = (var1) => {
+    let var2 = var1.sort((a, b) => a - b).reverse()
+    return var2
+}
+console.log(q5(sanDiegoSummerTemperatures)) // output: [82, 80, 79, 77, 76, 73, 72]
+console.log(q5(sanDiegoWinterTemperatures)) // output: [68, 67, 66, 66, 62, 59, 59]
+// I had to reference syllabus examples and outside resources to make this idea work, did not initially know how to sort an array of numbers.
```

Comment:
Resources are certainly our friend!  Just make sure to note in your pseudocode what is happening within the sort method, as there is some important logic happening there.